### PR TITLE
Initial commit - yaml from content

### PR DESCRIPTION
**Pull request by Shakedp**
## Status
In Progress

## Related Issues
https://github.com/demisto/etc/issues/17477

## Description
Shodan integration enhancement

## Required version of Demisto
5.0.0

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Integrations/Shodan/Shodan.yml [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+description: search engine for Internet-connected devices
+configuration:
+- display: Api Key
```
Comment (Shakedp) [inline Integrations/Shodan/Shodan.yml]: What about proxy paramter?

File: Integrations/Shodan/Shodan.yml [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  - name: search
+    arguments:
+    - name: query
+      required: true
+      default: true
+      description: 'Shodan search query. The provided string is used to search the
+        database of banners in Shodan, with the additional option to provide filters
+        inside the search query using a "filter:value" format. For example, the following
+        search query would find Apache webservers located in Germany: "apache country:DE"'
+    - name: facets
```
Comment (Shakedp) [inline Integrations/Shodan/Shodan.yml]: Isn't it an array?

File: Integrations/Shodan/Shodan.yml [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+      description: A comma-separated list of properties to get summary information
+        on. Property names can also be in the format of "property:count", where "count"
+        is the number of facets that will be returned for a property (i.e. "country:100"
+        to get the top 100 countries for a search query)
+    - name: page
+      description: Result page number to be fetched. Each page contains up to 100 results.
+      defaultValue: "1"
+    description: Search Shodan using the same query syntax as the website and use
+      facets to get summary information for different properties.
```
Comment (Shakedp) [inline Integrations/Shodan/Shodan.yml]: no outputs? @Arsenikr

File: Integrations/Shodan/Shodan.yml [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+      description: IP Address
+    - contextPath: IP.Geo.Country
+      description: Country of given IP
+    - contextPath: IP.Geo.Description
+      description: 'Description of location '
+    - contextPath: IP.Geo.Location
+      description: Latitude and longitude of given IP
+    - contextPath: IP.Hostname
+      description: Hostname
+    - contextPath: Shodan.IP.Tags
```
Comment (Shakedp) [inline Integrations/Shodan/Shodan.yml]: changed to Tag

File: Integrations/Shodan/Shodan.yml [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+    - contextPath: Shodan.IP.Longitude
+      description: ''
+      type: Number
+    - contextPath: Shodan.IP.LastUpdate
+      description: ''
+      type: Date
+    - contextPath: Shodan.IP.CountryName
+      description: ''
+      type: String
+    - contextPath: Shodan.IP.Address
```
Comment (Shakedp) [inline Integrations/Shodan/Shodan.yml]: not in the response, removing

File: Integrations/Shodan/Shodan.yml [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+      description: ''
+      type: String
+    - contextPath: Shodan.IP.Os
```
Comment (Shakedp) [inline Integrations/Shodan/Shodan.yml]: changing to OS

File: Integrations/Shodan/Shodan.yml [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+      description: ''
+      type: String
+    - contextPath: Shodan.IP.Ports
```
Comment (Shakedp) [inline Integrations/Shodan/Shodan.yml]: changing to Port

File: Integrations/Shodan/Shodan.yml [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+    - contextPath: Shodan.IP.Tags
+      description: ''
+      type: String
+    - contextPath: Shodan.IP.Latitude
+      description: ''
+      type: Number
+    - contextPath: Shodan.IP.Org
+      description: ''
+      type: String
+    - contextPath: Shodan.IP.Asn
```
Comment (Shakedp) [inline Integrations/Shodan/Shodan.yml]: ASN

File: Integrations/Shodan/Shodan.yml [hunk 9]
```diff
@@ -0,0 +1,235 @@
...
+      description: ''
+      type: String
+    - contextPath: Shodan.IP.Isp
```
Comment (Shakedp) [inline Integrations/Shodan/Shodan.yml]: ISP

File: Integrations/Shodan/Shodan.yml (hunk 1 above)
Comment (Arsenikr) [reply Integrations/Shodan/Shodan.yml]: we have proxy parameter there - L19

File: Integrations/Shodan/Shodan.yml (hunk 3 above)
Comment (Arsenikr) [reply Integrations/Shodan/Shodan.yml]: will work on context for the search command

**Pull request closed without merging**